guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc

```diff
@@ -99,9 +99,10 @@ include::../common/modules/ref_post-upgrade.adoc[leveloffset=+2]
 [[following_the_progress_of_the_upgrade]]
 == Following the Progress of the Upgrade
 
-Because of the lengthy upgrade time, use a utility such as `screen` to suspend and reattach a communication session.
+ifdef::foreman-el,katello,satellite[]
+Because of the lengthy upgrade time, use a utility such as `tmux` to suspend and reattach a communication session.
 You can then check the upgrade progress without staying connected to the command shell continuously.
-For more information about using the screen command, see link:https://access.redhat.com/articles/5247[How do I use the screen command?] article in the _Red{nbsp}Hat Knowledge{nbsp}Base_.
-For more information, see the `screen` manual page.
+For more information, see the `tmux` manual page.
+endif::[]
```

Comment:
If you remove the link to redhat.com, please remove the whole "ifdef" directive in line 102 and 106.